.github/ISSUE_TEMPLATE/uso_de_issue.yml

```diff
@@ -6,10 +6,18 @@ body:
     attributes:
       value: |
         # Plantilla para para que los issues que se abran sigan las buenas prácticas
-        Gracias por usar esta plantilla. Los issues son la fuente de verdad de nuestro proyecto.
```

Comment:
No es que esté mal quitar esto, pero está fuera de ámbito en este PR. En cada PR debéis ceñiros a lo que se pide en el PR, o corres el riesgo de comenzar una cascada de cambios sin relación o simplemente bloquear un PR por errores en algo que no esté realmente relacionado con el issue.